h2o-py/h2o/model/word_embedding.py

```diff
@@ -39,4 +40,12 @@ def transform(self, words, aggregate_method):
         :returns: the approximate reconstruction of the training data.
         """
         j = h2o.api("GET /3/Word2VecTransform", data={'model': self.model_id, 'words_frame': words.frame_id, 'aggregate_method': aggregate_method})
-        return h2o.get_frame(j["vectors_frame"]["name"])
\ No newline at end of file
+        return h2o.get_frame(j["vectors_frame"]["name"])
+
+    def to_frame(self):
+        """
+        Converts a given word2vec model into H2OFrame.
+
+        :returns: a frame representing learned word embeddings.
+        """
+        return h2o.H2OFrame._expr(expr=ExprNode("word2vec.to.frame", self))
```

Comment:
What will happened if the backend rapids is not available? Should be check for that at client level and report it? I am planning to expose capability API as part of modularization (https://0xdata.atlassian.net/browse/PUBDEV-4280)